src/servicex_did_finder_lib/servicex_adaptor.py

```diff
@@ -61,14 +62,17 @@ def post_status_update(self, status_msg, severity="info"):
             self.logger.error(f'After {attempts} tries, failed to send ServiceX App a status '
                               f'message: {str(status_msg)} - Ignoring error.')
 
+    def _prefix_file(self, file_path):
+        return file_path if not self.file_prefix else self.file_prefix+file_path
```

Comment:
This particular prefix only works with the `root://` protocol, and some return an `http`. We have no rules in place yet that a DID finder should return just one or the other.

Should we have a safety in here? Call this `root_prefix` or something like that, and only apply for a root:// string?